<launch>

  <arg name="rviz" default="true" />


  <node pkg="livox_mapping" type="livox_mapping_case" name="livox_mapping_case" output="screen">
  <param name="map_file_path" type="string" value=" " />
  </node>

  <group if="$(arg rviz)">
    <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find livox_mapping)/rviz_cfg/livox_mapping.rviz" />
  </group>




</launch>

